### ISSUE TITLE
Mousewheel issue: add a threshold to better handle inertia

### ISSUE THREAD

**Issue by timothyhague** [nolimits4web/swiper#3719]
This is a (multiple allowed):
* [ ] bug
* [x] enhancement
* [ ] feature-discussion (RFC)

* Swiper Version: 6.0.4
* Platform/Target and Browser Versions: All

### What you did
I'm using swiper on my company's site.

I've added the Mousewheel module, and set params to include:
```
direction: 'horizontal',
mousewheel: {
    forceToAxis: true,
  },
slidesPerView: 3,
slidesPerGroup:3,
spaceBetween: 20,
```

### Behavior and suggested improvement

It is working quite well, however, when I scroll, it often jumps two pages of slides instead of one, due to inertia of the scroll. It's particularly noticeable with a mac trackpad.

To improve, I suggest adding a threshold param to the mousewheel module, so that only scroll events over that delta threshold will be acknowledged. Something like:
https://github.com/timothyhague/swiper/commit/bca9553696b823bb663dd39ca77f3e7dfa2ca1df

**Comment by CapablancaYEHA** [comment]
@timothyhague 
can you please tell, what the trick behind you `mousewheel` prop working at all in the first place?
Do you have special css styles on container or smth ? I try in React btw.

I just use html layout from MouseWheel control Demo page + props from there (or your props) and it doesnt work.

**Comment by timothyhague** [comment]
> @timothyhague
> can you please tell, what the trick behind you `mousewheel` prop working at all in the first place?
> Do you have special css styles on container or smth ? I try in React btw.
> 
> I just use html layout from MouseWheel control Demo page + props from there (or your props) and it doesnt work.
> 
> same for Vertical mousewheel - never works

Hello,

I'm using React too. If I don't set anything for `preventSwipeThreshold`, all continues to function as before. Should have no effect. If you want, then you can set a value for it like this:

```
mousewheel: {
    forceToAxis: true,
    preventSwipeThreshold: 120,
  },
```
and this will mean you have to scroll at a higher velocity for it to be acknowledged.

**Comment by timothyhague** [comment]
I just modified the pr so there are two new params on mousewheel:

```
mousewheel: {
    preventSwipeThresholdDelta: 120,
    preventSwipeThresholdTime: 1200,
},
```

Which allows you to configure both time and delta thresholds:
https://github.com/nolimits4web/swiper/pull/3720

**Comment by CapablancaYEHA** [comment]
@timothyhague i mean slighltly different
I use Gatsby in React, and mousewheel doesnt work at all.
You dont face similar issues ?

**Comment by monika2345** [comment]
Hi,

Can anyone share the complete source code of mouse wheels slider or share any vertical slider code because am not able to find the below file which is required to run slider like CSS file and JS.


  <link rel="stylesheet" href="../package/swiper-bundle.min.css">

<script src="../package/swiper-bundle.min.js"></script>

I also use stack code they are using web reference but it also not working please help me

**Comment by nolimits4web** [comment]
Fixed by #3720
